src/stores/ProjectsInstitutionStore.js

```diff
@@ -40,6 +41,18 @@ export const useProjectInstitutionStore = defineStore({
       return false
     },
 
+    async getInstitutionById(projectId, institutionId) {
+      const url = `${
+        import.meta.env.VITE_API_URL
+      }/projects/${projectId}/institutions/add`
```

Comment:
The endpoint is called `add` but fetches the institution. This will confuse future SWEs who call this method and think that this will add an institution to a project.